samples/client-sdk/getting-started-daily-js/index.js

```diff
@@ -139,13 +158,49 @@ class DailyCallManager {
 
   /**
    * Adds a <video> element to the DOM and attaches the video track to it.
+   * Also displays the session ID of the participant as an overlay.
+   * If the video track is stopped, keeps the placeholder.
    * @param {Object} event - The track event object containing the video track.
    */
   displayVideo(event) {
-    const videosDiv = document.getElementById('videos');
-    const videoEl = document.createElement('video');
-    videosDiv.appendChild(videoEl);
-    videoEl.style.width = '100%';
+    const participantId = event.participant.session_id; // Assume this is how we get the participant ID
```

Comment:
Tempted to suggest just cutting this comment as for someone new to Daily and not yet thinking about _other_ ways to get the participant ID it might be a little confusing.

Reply:
Whoops! That's my own comment left over. 🙈 Removing.